wcfsetup/install/files/lib/system/language/event/PhraseChanged.class.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace wcf\system\language\event;
+
+use wcf\data\language\Language;
+use wcf\system\event\IEvent;
+
+/**
+ * Indicates that a phrase has been modified by the user.
+ *
+ * @author Alexander Ebert
+ * @copyright 2001-2022 WoltLab GmbH
+ * @license GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @package WoltLabSuite\Core\System\Language\Event
+ * @since 6.0
+ */
+final class PhraseChanged implements IEvent
+{
+    public readonly Language $language;
+    public readonly string $name;
+
+    public function __construct(Language $language, string $name)
+    {
+        $this->language = $language;
+        $this->name = $name;
+    }
```

Comment:
Something fundamental that needs to be discussed:

I've generally avoid storing the original DBO in Value Objects (and thus events) allow for a cleaner serialization.

see:
https://github.com/WoltLab/WCF/blob/87c040a366b227f321c3fc9e747391a9e489806b/wcfsetup/install/files/lib/system/user/authentication/event/UserLoggedIn.class.php#L29

and also:

https://github.com/WoltLab/WCF/blob/87c040a366b227f321c3fc9e747391a9e489806b/wcfsetup/install/files/lib/system/email/Mailbox.class.php#L51-L55